android/BOINC/.idea/codeStyles/Project.xml

```diff
@@ -1,30 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <Objective-C-extensions>
```

Comment:
Please move changes in this file into separate pull request because they are no belongs to the main functionality that is under implementation in this pull request